CONTRIBUTING.md

```diff
@@ -35,8 +35,47 @@ just a few small guidelines you need to follow.
         1.  For Google-authored source files, paste the Apache header text in
             the comments at the top. (If you're using different license,
             include the full text of that license.)
+1.  Place an executable called `pipeline` on the root of your solution folder
+    which builds, deploy and tests your solution. This file will be executed by
+    our automation daily.
 1.  Submit a pull request.
 
+## DevRel Automation
+
+Apigee DevRel uses automation that runs daily to ensure all solutions build successfully with all tests passing. It is recommended for the contributors to run the same checks locally at least once before every pull request to ensure build won't fail in our automation.
+
+In order to run this process locally for your solution:
+
+```
+npm run build-pipeline-runner
+npm run pipeline -- <path-to-your-solution-folder>
+
+E.g. npm run pipeline -- ./demos/hello-world
+```
+
+In order to run this process locally for all solutions within DevRel, execute the following commands:
```

Comment:
This requires docker and you must be in the docker group (not requiring sudo access)

Reply:
in our particular linux environment, yes. Not in any of the others... Again teaching how to run docker (with or without sudo) might be outside our intent here.